src/main/scala/viper/silver/cfg/silver/SilverCfg.scala

```diff
@@ -82,6 +82,65 @@ class SilverCfg(val blocks: Seq[SilverBlock], val edges: Seq[SilverEdge], val en
                     entry: SilverBlock = entry,
                     exit: Option[SilverBlock] = exit): SilverCfg =
     SilverCfg(blocks, edges, entry, exit)
+
+  private def findJoinPoint(queueInit: Iterable[SilverBlock],
```

Comment:
Please add (brief) documentation
* overall purpose/functionality
* documentation of parameters and results
* high-level summary of the algorithm